dex-it/src/test/scala/com/wavesplatform/it/sync/api/ws/PingPongTestSuite.scala

```diff
@@ -99,17 +99,31 @@ class PingPongTestSuite extends MatcherSuiteBase with HasWebSockets {
       Thread.sleep(pongTimeout - 0.2.second)
 
       Seq(wsac1, wsac2).foreach { conn =>
-        conn.isClosed shouldBe false
         conn.getPingsBuffer should have size 3
+        conn.isClosed shouldBe false
       }
 
       Thread.sleep(0.1.second)
       eventually {
         Seq(wsac1, wsac2).foreach { conn =>
-          conn.isClosed shouldBe true
           conn.getPingsBuffer.size should (be >= 3 and be <= 4)
+          conn.isClosed shouldBe true
         }
       }
     }
+
+    "by signed connection lifetime expiration, if it < max-connection-lifetime" in {
+      val wsac = mkWsAuthenticatedConnection(alice, dex1, keepAlive = false, connectionLifetime = 1.5.seconds)
+      wsac.isClosed shouldBe false
+
+      Thread.sleep(1.4.seconds)
+      wsac.isClosed shouldBe false
+
+      Thread.sleep(0.1.second)
+      eventually {
```

Comment:
I think using `eventually` here is not right, because eventually wait for 30 or 40 seconds. it could be false positive